evm/vm/state.py

```diff
@@ -287,24 +251,44 @@ def get_uncle_reward(block_number, uncle):
     def get_nephew_reward(cls):
         raise NotImplementedError("Must be implemented by subclasses")
 
+    def get_transaction_executor(self, transaction):
+        vm_state = self
```

Comment:
This is odd to me.  can we just use `self` on the next line?

Reply:
Sure. I added it when there were more arguments passed in, which made it look weird to me:

```
return self.transaction_executor(self, transaction)
```